dateparser/languages/language.py

```diff
@@ -24,40 +26,51 @@ def __init__(self, shortname, language_info):
             if isinstance(value, int):
                 simplification[key] = str(value)
 
+        self._cached = self
```

Comment:
I did that because we generate a new instance of Language whenever a new skip token is added and identified like [here](https://github.com/scrapinghub/dateparser/pull/86/files#diff-70e80316a9eb316ceb7f7f1c8f63d152R42). Is there a better practice?